tests/api/common.c

```diff
@@ -0,0 +1,81 @@
+#include "common.h"
+#include "accelerator/cli_info.h"
+#include "stdlib.h"
+
+static struct ta_cli_argument_s driver_cli_arguments_g[] = {
+    {"hash", required_argument, NULL, 's', "testing transaction hashes"},
+    {"tag", required_argument, NULL, 'g', "testing transaction tag"},
```

Comment:
Why `g`?

Reply:
I used `s` in `hash`, and `g` in `tag`. 
Or I should set the `ta_cli_arg_value_e` of these two commands from the maximun number?

Reply:
How about `t` for `tag` and `h` for `hash`?

Reply:
`h` has been used for `help`. I think using `t` for tag.

Reply:
Or, use `H` for `hash`.

Reply:
For consistency, I will use `T` for tag.